docs/docs/cli-reference/grafloki-stop.md

```diff
@@ -0,0 +1,13 @@
+---
+title: grafloki stop
+sidebar_label: grafloki stop
+slug: /grafloki-stop
+---
+
+To stop a Grafana/Loki instance in Docker or K8s cluster, run:
```

Comment:
Perhaps useful to link to `grafloki start` just so people can get there easily if they come across this page